### DISCUSSION TITLE
How can I find and print the IGD values of every evolutionary generation?

### DISCUSSION THREAD

**Discussion by nsygtc** [anyoptimization/pymoo#553] [Q&A]
Hi, I am doing some research on multi-objective optimization, and I want to get the IGD value of every generation to observe the variety of experimental IGD values. I can just observe these data by setting the parameters "verbose=True, save_history=True", but I can't get the IGD values to process further. So how can I print the IGD values of every evolutionary generation? Thank you very much.

**Comment by blankjul** [comment]
Checkout the getting start guide which analyzes the convergence as well:
https://pymoo.org/getting_started/part_4.html

**Reply by nsygtc** [reply]
Thank you very much!
